test/integration/validate-tts.js

```diff
@@ -0,0 +1,51 @@
+const fs = require('fs')
+const path = require('path')
+const { setupSupertestServer } = require('../utils')
+
+const server = setupSupertestServer({
+  live: true,
+  dataBrowserPath: 'default',
+  root: path.join(__dirname, '../resources'),
```

Comment:
I'd have split it to `path.join(__dirname, '..', 'resources')`, but if it works everywhere …